src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java

```diff
@@ -329,6 +330,17 @@ public String value() {
   public static final String OVERRIDE_ERRORS_TOLERANCE_DOC =
       "Use this property if you would like to configure the connector's error handling behavior differently from the Connect framework's.";
 
+  public static final String ERRORS_RETRIES_COUNT_DISPLAY = "Error Retries Count";
```

Comment:
Do we need mongo specific overrides for the existing kafka retry configurations? There already is connect configuration properties for [`errors.retry.timeout`](https://docs.confluent.io/platform/current/installation/configuration/connect/sink-connect-configs.html#errors-retry-timeout) and `errors.retry.delay.max.ms`. Should these be used instead?

See [KAFKA-215](https://jira.mongodb.org/browse/KAFKA-215) for the reasoning behind the addition of `mongo.errors.tolerance`.

Reply:
I do think we need these overrides because kafka retry is probably still meaningful for some other scenarios like startup.